charts/azure-js/templates/deployment.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{.Release.Name}}-js
```

Comment:
drop the -js suffix in all occurrences in the file

Reply:
done